symfit/core/printing.py

```diff
@@ -1,9 +1,9 @@
 from sympy.printing.pycode import NumPyPrinter
 from sympy.printing.codeprinter import CodePrinter
 
-#########################################################
-# Monkeypatch the printer until this is merged upstream #
-#########################################################
+##########################################################
+# Monkeypatch the printers until this is merged upstream #
+##########################################################
```

Comment:
It should probably be mentioned in the docs that we monkeypatch some printers which may cause strange results for users that import symfit, but expect sympy printing elsewhere in their code. Different PR.